tests/test_hitlog.py

```diff
@@ -0,0 +1,32 @@
+# SPDX-FileCopyrightText: Portions Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import contextlib
+import os
+import pytest
+
+import garak
+import garak.cli
+
+
+CODEPATH_PREFIX = "_garak_test_hitlog_codepath"
+
+
+def test_hitlog_codepath():
+
+    args = f"-m test.Blank --report_prefix {CODEPATH_PREFIX} -p test.Test -d always.Fail".split()
+    garak.cli.main(args)
+    assert os.path.isfile(f"{CODEPATH_PREFIX}.hitlog.jsonl")
+
+
+@pytest.fixture(scope="session", autouse=True)
+def cleanup(request):
+    """Cleanup a testing directory once we are finished."""
+
+    def remove_codepath_reports():
+        with contextlib.suppress(FileNotFoundError):
+            os.remove(f"{CODEPATH_PREFIX}.report.jsonl")
+            os.remove(f"{CODEPATH_PREFIX}.report.html")
+            os.remove(f"{CODEPATH_PREFIX}.hitlog.jsonl")
+
+    request.addfinalizer(remove_codepath_reports)
```

Comment:
This is not needed as the `config_report_cleanup` fixture in `tests/conftests.py` is already used by the tests in this file. I have validated the `config_report_cleanup` is applied when running this test.

```suggestion
```